tests/end-to-end/tests/cards.cy.ts

```diff
@@ -0,0 +1,148 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { selectors, testIds } from '../support/consts';
+
+const paths = {
+  cardsList:
+    'ComponentOverlay_rootcontainer/maincontainer/pagesection/cardslist',
+  cardItem:
+    'ComponentOverlay_rootcontainer/maincontainer/pagesection/cardslist/card1'
+};
+
+describe('Cards lists and card item component', function () {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('render correctly in preview mode', function () {
+    cy.visit('/content/howlite-test/pages/Cards.html');
+
+    cy.percySnapshotWithAuth('Cards preview');
+
+    cy.get('section:nth-of-type(2) div').should('have.class', 'glide--slider');
+    cy.contains('Third card item').move({ deltaX: 1500, deltaY: 0 });
+    cy.contains('Last card item').should('be.visible');
+
+    cy.get('section:last-of-type ul').should('have.class', 'hl-grid-lg-2');
+  });
+
+  it('render correctly in edit mode', function () {
+    cy.visit(
+      '/apps/websight/index.html/content/howlite-test/pages/Cards::editor'
+    );
+
+    testCardsList();
+    testCardItem();
+  });
+
+  function testCardsList() {
+    cy.percySnapshotWithAuth('Cards list editor');
+
+    cy.intercept(
+      'POST',
+      '**/pagesection/cardslist.websight-dialogs-service.save-properties.action'
+    ).as('saveProperties');
+
+    cy.getByTestId(paths.cardsList)
+      .click({ force: true })
+      .find(selectors.overlayName)
+      .should('have.text', 'Cards List');
+
+    cy.getByTestId(testIds.editIcon).click();
+
+    cy.percySnapshotWithAuth('Cards list dialog');
+
+    cy.getByTestId('Input_Displayasaslider--checkbox-label').click();
+    cy.getByTestId('RadioElement_h3').click();
+    cy.getByTestId('RadioElement_hl-title__heading--size-3').click();
+    cy.get('#undefined-1').click();
+    cy.get('div[id^="classes-uid"]').click();
+    cy.contains('Numbered').click({ force: true });
+
+    cy.getByTestId(testIds.dialogSubmitButton).click();
+
+    cy.wait('@saveProperties');
+
+    cy.request(
+      '/content/howlite-test/pages/Cards/jcr:content/rootcontainer/maincontainer/pagesection/cardslist.json'
+    )
+      .its('body')
+      .should('deep.eq', {
+        'sling:resourceType': 'howlite/components/cardslist',
+        'jcr:primaryType': 'nt:unstructured',
+        isSlider: 'true',
+        classes: 'hl-cards-list--numbered',
+        headingLevel: 'h3',
+        headingSize: 'hl-title__heading--size-3'
```

Comment:
I think we should add a check also "Items per row" on every breakpoint (let's change it to something else on every breakpoint)